less/forum/Post.less

```diff
@@ -116,6 +116,14 @@
     border-bottom: 2px dotted @body-bg;
     margin: 1em 0;
   }
+  code {
+    font-family: source-code-pro, Monaco, Consolas, "Courier New", monospace;
+    padding: 2px 6px;
+    background: #60646D;
+    color: #256FC7;
```

Comment:
Ah, something I forgot till now. You should use one of the existing color variables probably instead of copying the value here... that way, theming from the admin panel will be easier later on.